blockwork/build/caching.py

```diff
@@ -337,14 +410,22 @@ def drop_value(self, key: str) -> bool:
         return self.drop_item(key)
 
 
-    def fetch_value(self, key: str) -> Optional[str]:
+    def fetch_value(self, key: str, peek: bool=False) -> Optional[str]:
         '''
         Fetch a string value from the store by key. Return None if not present.
```

Comment:
Could really do with some markup of parameters and what they do

Reply:
Added params to the cache docstrings